src/main/java/rx/Observable.java

```diff
@@ -1069,6 +1069,11 @@ public void call(Subscriber<? super R> o) {
         return create(new OnSubscribeDefer<T>(observableFactory));
     }
 
+    @Experimental
+    public static <T> Observable<T> lazy(Func0<? extends T> lazyFunc) {
```

Comment:
I'd consider Callable to allow IO operations without the need to do try-catch. An overload won't work due to dynamic languages.